### DISCUSSION TITLE
Is it down???

### DISCUSSION THREAD

**Discussion by ghost** [Nerimity/nerimity-web#87] [Q&A]
Is nerimity down? 
![Screenshot 2024-09-02 9 44 37 AM](https://github.com/user-attachments/assets/9b01951d-faaa-4a72-ab10-93d2d196e7d3)

**Comment by SupertigerDev** [comment]
Unfortunately yes, my vps provider contabo is issues https://contabo-status.com

**Reply by ghost** [reply]
Ah okay, Hope it's fixed!

**Reply by SupertigerDev** [reply]
It's back

**Reply by ghost** [reply]
W :3
